.gitignore

```diff
@@ -6,3 +6,7 @@
 
 # Ignore Gradle build output directory
 build
+
+benchfiles/
+utilities/src/main/java/ru/study21/jcsv/xxl/tmp/
+utilities/tmp/
```

Comment:
Are you really sure that it is gradle that produced such directories ?

Reply:
I guess it is result of using temporary directory creation API.
I would be better to solve such kind of problem in another way.
For example, define some default configuration to store temporary file in other place.
Or remove it on shutdown.

Reply:
No, these are just my files I used for temporary reasons inside this package and didn't want to commit

Reply:
So it is your local problem.
And change of repo's .gitignore is action dealing with global scope :)